js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties

```diff
@@ -177,3 +202,438 @@ category=Kategorie
+createClientScope=Client-scope erstellen
+searchForClientScope=Suche nach Client-scope
+changeTypeTo=Zugeordneten Typ ändern
+clientsExplain=Clients sind Anwendungen und Dienste, die die Authentifizierung eines Benutzers anfordern können.
+clientsList=Client Liste
+createClient=Client erstellen
+importClient=Client importieren
+searchForClient=Suche nach Client
+initialAccessToken=Initiale Access-Token
+noTokens=Keine initialen Access-Token vorhanden
+noTokensInstructions=Sie haben noch keine initiale Access-Token erstellt. Erstellen Sie ein Access-Token, indem Sie auf "Erstellen" klicken.
+clientRegistration=Client Registrierung
+anonymousAccessPolicies=Anonyme Zugriffsrichtlinien
+searchClientRegistration=Suche nach Richtlinie
+anonymousAccessPoliciesHelp=Diese Richtlinien werden verwendet, wenn der Client Registration Service durch eine nicht authentifizierte Anfrage aufgerufen wird. Dies bedeutet, dass die Anfrage weder ein Initial Access Token noch ein Bearer Token enthält.
+authenticatedAccessPolicies=Richtlinien für authentifizierten Zugriff
+authenticatedAccessPoliciesHelp=Diese Richtlinien werden verwendet, wenn der Client Registration Service durch eine authentifizierte Anfrage aufgerufen wird. Dies bedeutet, dass die Anfrage ein Initial Access Token oder Bearer Token enthält.
+createClientPolicy=Client-Richtlinie erstellen
+realmInfo=Realm Informationen
+welcomeTabTitle=Willkommen
+welcomeTo=Willkommen zu {{realmDisplayInfo}}
+welcomeText=Keycloak bietet Benutzerföderation, starke Authentifizierung, Benutzerverwaltung, feinkörnige Autorisierung und mehr. Fügen Sie Authentifizierung zu Anwendungen und sicheren Diensten mit minimalem Aufwand hinzu. Es gibt keine Notwendigkeit, sich mit der Speicherung von Benutzern oder der Authentifizierung von Benutzern in der Anwendung zu beschäftigen, dies erledigt Keycloak.
+viewDocumentation=Siehe Dokumentation
+viewGuides=Guides ansehen
+joinCommunity=Community beitreten
+readBlog=Blog lesen
+memory=Speicher
+totalMemory=Gesamter Speicher
+freeMemory=Freier Speicher
+usedMemory=Verwendeter Speicher
+enabledFeatures=Aktivierte Features
+infoEnabledFeatures=Zeigt eine aktivierte Vorschau und experimentelle Funktionen an.
+disabledFeatures=Deaktivierte features
+infoDisabledFeatures=Zeigt alle deaktivierten Funktionen an.
+acrToLoAMapping=ACR zu LoA Zuordnungen
+acrToLoAMappingHelp=Definieren Sie, welcher ACR-Wert (Authentication Context Class Reference) welchem LoA-Wert (Level of Authentication) zugeordnet ist. Die ACR kann ein beliebiger Wert sein, während die LoA numerisch sein muss.
+missingAttributes=Es sind noch keine {{label}} definiert worden. Klicken Sie auf die untere Schaltfläche, um {{label}} hinzuzufügen, Schlüssel und Wert sind für ein Schlüsselpaar erforderlich.
+addProvider=Anbieter hinzufügen
+providerDescription=Anbieter Beschreibung
+xFrameOptionsHelp=Der Standardwert verhindert, dass Seiten von Iframes, die nicht von der Website stammen, eingebunden werden <1>Mehr erfahren</1>
```

Comment:
I think Iframes should be written iframe but this is probably more opinionated https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe :)

Reply:
I agree!